Add twenty-seven abilities

anblusis opened the pull request:
## 개요
27개의 능력을 추가했습니다. 

 - alchemy (연금술)
 - architect (건축)
 - band (악단)
 - barrier (방벽)
 - boxing (권투)
 - burning mana (마나 태우기)
 - dream world (꿈나라)
 - drone (드론)
 - electric shock wave (전격파)
 - electrician (전기술사)
 - eruption (분화)
 - falconer (매잡이)
 - flamethrower (화염 방사)
 - guardian (수호자)
 - hoe spin (스핀)
 - magician (마술)
 - mediation (중개)
 - multi tonic (종합영양제)
 - paddywhack (노발대발)
 - pumpkin ghost (호박 유령)
 - random egg (이스터에그)
 - shackle (속박)
 - sleep (수면)
 - sniping (저격)
 - tolerance (포용)
 - wind shot (장풍)
 - worse matter (병상첨병)

 ## 스크린샷
 - alchemy (연금술)
![image](https://user-images.githubusercontent.com/101804845/198880400-c51f428f-d2d3-49c5-b63f-5af277d3fe1b.png)

 - architect (건축)
![image](https://user-images.githubusercontent.com/101804845/218969770-7a63b184-55c2-4985-9225-e65af217446c.png)

 - band (악단)
![image](https://user-images.githubusercontent.com/101804845/200160655-13a40f2d-901e-44c1-8ee5-e873434036f3.png)

 - barrier (방벽)
![image](https://user-images.githubusercontent.com/101804845/198880464-1506ddca-ce40-4aac-a033-678944037ffc.png)

 - boxing (권투)
![image](https://user-images.githubusercontent.com/101804845/198880513-22552d12-a4ce-41a7-83dc-fea1ccf305db.png)

 - burning mana (마나 태우기)
![image](https://user-images.githubusercontent.com/101804845/198880573-ceb64c22-635c-4207-a0fb-473072de1e30.png)

 - dream world (꿈나라)
![image](https://user-images.githubusercontent.com/101804845/198880645-58182778-940d-4df0-a314-d45fd387cd12.png)

 - drone (드론)
![image](https://user-images.githubusercontent.com/101804845/198880666-c967ff69-2c8b-4162-a18d-fcc0e90ea6c9.png)

- electric shock wave (전격파)
![image](https://user-images.githubusercontent.com/101804845/212468819-4f497ae8-e929-4c3c-bd95-217b70af3f42.png)

 - electrician (전기술사)
![image](https://user-images.githubusercontent.com/101804845/198880756-04a34584-934d-459e-8a9b-55e5c05b43f6.png)

 - eruption (분화)
![image](https://user-images.githubusercontent.com/101804845/198880785-aa3ca2da-d98f-447a-9271-308b7e4ff90f.png)

 - falconer (매잡이)
![image](https://user-images.githubusercontent.com/101804845/212468856-b502a278-288e-44bd-9e5a-ae77c0efb13f.png)

 - flamethrower (화염 방사)
![image](https://user-images.githubusercontent.com/101804845/198880796-7a93220b-a5c9-4ffc-b435-30c60f5ca289.png)

 - guardian (수호자)
![image](https://user-images.githubusercontent.com/101804845/198880807-c7a9ae0b-85d7-4cd3-bfed-29cd5e80befa.png)

- hoe spin (스핀)
![image](https://user-images.githubusercontent.com/101804845/210139728-ec179808-b9ea-4bf0-8fd9-90af9534cd19.png)

 - magician (마술)
![image](https://user-images.githubusercontent.com/101804845/198880823-1c288f23-5589-42d0-9222-fea41f25b8b3.png)

 - mediation (중개)
![image](https://user-images.githubusercontent.com/101804845/198880845-89d74648-661a-4cd9-945b-156ad5679610.png)

 - multi tonic (종합영양제)
![image](https://user-images.githubusercontent.com/101804845/198880868-418e34e9-4ecf-489a-b551-6752793b745e.png)

- paddywhack (노발대발)
![image](https://user-images.githubusercontent.com/101804845/202172866-10af3765-6f0b-4bf5-8477-de65668e0e41.png)

- pumpkin ghost (호박 유령)
![image](https://github.com/monun/psychics/assets/101804845/9d3d362d-66fc-4b80-9bb1-18da43584ae1)

 - random egg (이스터에그)
![image](https://user-images.githubusercontent.com/101804845/198880895-76a07952-b291-4cb0-917c-d68a6b04dafe.png)

 - shackle (속박)
![image](https://user-images.githubusercontent.com/101804845/212468916-410692c4-a577-4d3e-9eec-a0b5319a9243.png)

 - sleep (수면)
![image](https://user-images.githubusercontent.com/101804845/198880910-8d075a54-c601-42e1-8bdf-83a0b33d4575.png)

 - sniping (저격)
![image](https://user-images.githubusercontent.com/101804845/198880930-9fb80c72-1ef6-4ce1-a431-8eafb5641f02.png)

 - tolerance (포용)
![image](https://user-images.githubusercontent.com/101804845/218969545-5055a87f-5f4a-4988-8fb7-ba93befbcf9a.png)

 - wind shot (장풍)
![image](https://user-images.githubusercontent.com/101804845/198880945-babe3488-8829-415e-9027-2a3d3dd9001f.png)

 - worse matter (병상첨병)
![image](https://user-images.githubusercontent.com/101804845/198880625-f385beec-358f-410a-b9a9-e26c911d3bde.png)

anblusis committed:
Add sixteen abilities
-barrier
-boxing
-burning mana
-dream world
-drone
-eruption
-flamethrower
-guardian
-magician
-mediation
-multi tonic
-random egg
-sleep
-sniping
-wind shot
-worse matter

anblusis committed:
Add alchemy and edit 'sniping' remark

anblusis committed:
Add ability 'electrician'

anblusis committed:
Minor change in 'electrician'

anblusis committed:
Add ability 'band' and minor changes

anblusis committed:
Add ability 'paddywhack' + a minor change in ability 'band'

anblusis committed:
Add ability 'hoe spin' and fix a small bug in ability 'band'

anblusis committed:
Add three abilities
-electric-shock-wave
-falconer
-shackle
Edit code in ability 'guardian'

anblusis committed:
Fix a bug in ability "drone"

anblusis committed:
Add ability 'blast ball' and fix codes in four abilities
Fixed abilities
-alchemy
-electric shock wave
-mediation
-multi tonic

anblusis committed:
fix a small bug in ability 'mediation'

anblusis committed:
fix a small bug in ability 'mediation', maybe.

anblusis committed:
Add two abilities and fix a bug in ability 'magician'
Added abilities
-architect
-tolerance

UYEOL32 commented:
이 능력들을 어떻게 추가 하나요
기본 샘플 처럼  서버 폴더에 jar 파일이랑 yml 파일 넣어서 하는거 같은데
jar 파일을 만드는 법을 못 찾겠어요

anblusis committed:
Add ability 'pumpkin ghost'

anblusis committed:
minor changes in various abilities
-blast-ball
-boxing
-dream world
-drone
-electric shock wave
-eruption
-falconer
-flamethrower
-guardian
-hoe spin
-magician
-mediation
-paddywhack
-pumpkin ghost
-shackle
-sniping
-wind shot

anblusis commented:
앞으로는 능력을 1.20.1 버전에서 만들 예정이기 때문에 PR을 닫겠습니다.
